ui/app/AppLayouts/Wallet/views/CollectiblesView.qml

```diff
@@ -58,6 +58,26 @@ ColumnLayout {
         readonly property var sourceModel: root.controller.sourceModel
         readonly property bool isLoading: root.isUpdating || root.isFetching
 
+        function toggleSortByDateIsDisabled(value) {
```

Comment:
```suggestion
        function setSortByDateIsDisabled(value) {
```
I think this name is more appropriate because `toggle` usually doesn't take any argument and just change the bool value to the opposite one.

Reply:
done